### ISSUE TITLE
make for golang.org package fails "Could not create git repository: import-orig: exit status 1"

### ISSUE THREAD

**Issue by hickford** [Debian/dh-make-golang#190]
`dh-make-golang make golang.org/x/oauth2` fails.

It looks like it tries to extract a non-existent tar archive.

```
2022/12/18 04:34:50 Starting "dh-make-golang v0.6.0 linux/amd64"
2022/12/18 04:34:50 Downloading "golang.org/x/oauth2/..."
2022/12/18 04:34:51 Determining upstream version number
2022/12/18 04:34:51 Found latest tag "v0.3.0"
2022/12/18 04:34:51 Latest tag "v0.3.0" matches master
2022/12/18 04:34:51 Package version is "0.3.0"
2022/12/18 04:34:51 Determining dependencies
2022/12/18 04:34:53 INFO: Hoster does not provide release tarball
2022/12/18 04:34:53 WARNING: A package called "golang-golang-x-oauth2-google-dev" is already in Debian! See https://tracker.debian.org/pkg/golang-golang-x-oauth2-google-dev
2022/12/18 04:34:53 Moving tempfile to "golang-golang-x-oauth2_0.3.0.orig.tar.gz"
2022/12/18 04:34:53 Adding remote "origin" with URL "git@salsa.debian.org:go-team/packages/golang-golang-x-oauth2.git"
2022/12/18 04:34:53 Adding remote "golang" with URL "https://go.googlesource.com/oauth2"
2022/12/18 04:34:53 Running "git fetch golang"
remote: Total 2166 (delta 835), reused 2166 (delta 835)
Receiving objects: 100% (2166/2166), 1.32 MiB | 3.33 MiB/s, done.
Resolving deltas: 100% (835/835), done.
From https://go.googlesource.com/oauth2
 * [new branch]      master     -> golang/master
 * [new tag]         v0.3.0     -> v0.3.0
 * [new tag]         v0.1.0     -> v0.1.0
 * [new tag]         v0.2.0     -> v0.2.0
tar: This does not look like a tar archive

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
gbp:error: Couldn't unpack '/home/matt/golang-golang-x-oauth2_0.3.0.orig.tar.gz': it exited with 2
2022/12/18 04:34:54 Could not create git repository: import-orig: exit status 1
```

**Comment by anthonyfok** [comment]
Duplicate of #180 

- #180

**Comment by anthonyfok** [comment]
Duplicate of #180

**Comment by anthonyfok** [comment]
Fixed in b851343f599fd4c1113e5ba9e10fedeb856fb027
